### ISSUE TITLE
Semantic Conventions for Health Check events

### ISSUE THREAD

**Issue by thompson-tomo** [open-telemetry/semantic-conventions#1106]
### Area(s)

area:new

### Is your change request related to a problem? Please describe.

I want to be able to have health checks of all my different components recorded in  a consistent manner and in the end they would be producing an event

### Describe the solution you'd like

I would like for the introduction of a new attribute group healthCheck which contains the following recommended attributes

- healthCheck,type -> string
- healthCheck.result -> string
- healthCheck.resultDescription -> string
- healthCheck.duration ->float (number of seconds)

Health checks are then able to leverage other semantic conventions ie database to provide information about the database check.

Note i have based it dotnet core health checks but we should check other platforms ie SpringBoot to see what how they model their health checks to ensure we have an agnostic manner to describe the health check

**Comment by MSNev** [comment]
I don't believe that is the domain of the event working group to define the event "definition".

**Comment by martialblog** [comment]
Hi, as discussed over in the issue 3148, I think this is a sensible proposal and I can see some use cases.

I agree that `state_check` might be a better base namespace.
